app/i18n/zh-cn/install.php

```diff
@@ -33,89 +33,89 @@
 		'already_installed' => '我们检测到 FreshRSS 已经安装！',
 		'cache' => array(
 			'nok' => '请检查 <em>./data/cache</em> 目录权限。HTTP 服务器必须有其写入权限。',
-			'ok' => 'cache 目录权限正常。',
+			'ok' => 'cache 目录权限正常',
 		),
 		'ctype' => array(
-			'nok' => '找不到字符类型检测库 (php-ctype) 。',
-			'ok' => '已找到字符类型检测库 (ctype) 。',
+			'nok' => '找不到字符类型检测库（php-ctype）',
+			'ok' => '已找到字符类型检测库（ctype）',
 		),
 		'curl' => array(
-			'nok' => '找不到 cURL 库 (php-curl package) 。',
-			'ok' => '已找到 cURL 库。',
+			'nok' => '找不到 cURL 库（php-curl）',
+			'ok' => '已找到 cURL 库',
 		),
 		'data' => array(
 			'nok' => '请检查 <em>./data</em> 目录权限。HTTP 服务器必须有其写入权限。',
-			'ok' => 'data 目录权限正常。',
+			'ok' => 'data 目录权限正常',
 		),
 		'dom' => array(
-			'nok' => '找不到用于浏览 DOM 的库 (php-xml) 。',
-			'ok' => '已找到用于浏览 DOM 的库。',
+			'nok' => '找不到用于浏览 DOM 的库（php-xml）',
+			'ok' => '已找到用于浏览 DOM 的库',
 		),
 		'favicons' => array(
 			'nok' => '请检查 <em>./data/favicons</em> 目录权限。HTTP 服务器必须有其写入权限。',
-			'ok' => 'favicons 目录权限正常。',
+			'ok' => 'favicons 目录权限正常',
 		),
 		'fileinfo' => array(
-			'nok' => '找不到 PHP fileinfo 库 (fileinfo) 。',
-			'ok' => '已找到 fileinfo 库。',
+			'nok' => '找不到 PHP fileinfo 库（fileinfo）',
+			'ok' => '已找到 fileinfo 库',
 		),
 		'http_referer' => array(
-			'nok' => '请检查你是否修改了 HTTP REFERER。',
-			'ok' => '你的 HTTP REFERER 已知且与服务器一致。',
+			'nok' => '请检查你是否修改了 HTTP REFERER',
+			'ok' => '你的 HTTP REFERER 已知且与服务器一致',
 		),
 		'json' => array(
-			'nok' => '找不到推荐的 JSON 解析库。',
-			'ok' => '已找到推荐的 JSON 解析库。',
+			'nok' => '找不到推荐的 JSON 解析库',
+			'ok' => '已找到推荐的 JSON 解析库',
 		),
 		'mbstring' => array(
-			'nok' => '找不到推荐的 Unicode 解析库 (mbstring)。',
-			'ok' => '已找到推荐的 Unicode 解析库 (mbstring)。',
+			'nok' => '找不到推荐的 Unicode 解析库（mbstring)',
+			'ok' => '已找到推荐的 Unicode 解析库（mbstring)',
 		),
 		'minz' => array(
-			'nok' => '找不到 Minz 框架。',
-			'ok' => '已找到 Minz 框架。',
+			'nok' => '找不到 Minz 框架',
+			'ok' => '已找到 Minz 框架',
 		),
 		'pcre' => array(
-			'nok' => '找不到正则表达式解析库 (php-pcre) 。',
-			'ok' => '已找到正则表达式解析库 (PCRE) 。',
+			'nok' => '找不到正则表达式解析库（php-pcre）',
+			'ok' => '已找到正则表达式解析库（PCRE）',
 		),
 		'pdo' => array(
-			'nok' => '找不到 PDO 或支持的驱动 (pdo_mysql, pdo_sqlite, pdo_pgsql) 。',
-			'ok' => '已找到 PDO 和支持的至少一种驱动 (pdo_mysql, pdo_sqlite, pdo_pgsql) 。',
+			'nok' => '找不到 PDO 或支持的驱动（pdo_mysql、pdo_sqlite、pdo_pgsql）',
+			'ok' => '已找到 PDO 和支持的至少一种驱动（pdo_mysql、pdo_sqlite、pdo_pgsql）',
 		),
 		'php' => array(
-			'nok' => '你的 PHP 版本为 %s，但 FreshRSS 最低需要 %s。',
-			'ok' => '你的 PHP 版本为 %s，与 FreshRSS 兼容。',
+			'nok' => '你的 PHP 版本为 %s，但 FreshRSS 最低需要 %s',
+			'ok' => '你的 PHP 版本为 %s，与 FreshRSS 兼容',
 		),
 		'users' => array(
 			'nok' => '请检查 <em>./data/users</em> 目录权限。HTTP 服务器必须有其写入权限。',
-			'ok' => 'users 目录权限正常。',
+			'ok' => 'users 目录权限正常',
 		),
 		'xml' => array(
-			'nok' => '找不到用于 XML 解析库。',
-			'ok' => '已找到 XML 解析库。',
+			'nok' => '找不到用于 XML 解析库',
+			'ok' => '已找到 XML 解析库',
 		),
 		'_' => '检查',
 	),
 	'conf' => array(
-		'ok' => '常规配置已保存。',
```

Comment:
Just curious, is that `。` just a regular `.` (from our Latin perspective)?

Reply:
Yes, that mean the end of a sentence. So it is used just like the ' . ' in Latin in most time.